astropy/table/_column_mixins.pyx

```diff
@@ -42,11 +43,11 @@ ctypedef object (*binaryfunc)(object, object)
 
 cdef extern from "Python.h":
     ctypedef struct PyMappingMethods:
-        binaryfunc mp_subscript
+        pass
```

Comment:
```suggestion
```

This looks unused I think. (Sorry the suggestion is in two parts... Github UI isn't working well for me)